profiler/src/ProfilerEngine/Datadog.Profiler.Native/FrameStore.h

```diff
@@ -13,6 +13,24 @@
 
 class IConfiguration;
 
+#define ARRAY_LEN(a) (sizeof(a) / sizeof((a)[0]))
+
+void StrAppend(char* buffer, const char* str, size_t& cchBuffer);
+
+void FixGenericSyntax(WCHAR* name);
+void FixGenericSyntax(char* name);
+
+PCCOR_SIGNATURE ParseByte(PCCOR_SIGNATURE pbSig, BYTE* pByte);
+PCCOR_SIGNATURE ParseElementType(
+    IMetaDataImport* pMDImport,
+    PCCOR_SIGNATURE signature,
+    ClassID* classTypeArgs,
+    ClassID* methodTypeArgs,
+    ULONG* elementType,
+    char* buffer,
+    size_t cchBuffer,
+    mdToken* typeToken);
+
```

Comment:
I'm wondering if we need to add this to the header file: they are free functions only used by FrameStore class.
I think we could remove them from the `interface` of the class/file.

Reply:
✅ agreed